src/ansiblelint/prerun.py

```diff
@@ -201,7 +201,9 @@ def prepare_environment(required_collections: Optional[Dict[str, str]] = None) -
 
     if required_collections:
         for name, min_version in required_collections.items():
-            install_collection(f"{name}:>={min_version}")
+            install_collection(
+                f"{name}:>={min_version}", f"{options.cache_dir}/collections"
```

Comment:
Is it obvious that at this point, `options.cache_dir is not None`?

Reply:
Good point! I happen to know it should not be None by now but that is not the right way to deal with that. I will add an extra condition for that.